ra2ce/graph/networks_utils.py

```diff
@@ -1131,8 +1132,154 @@ def simplify_graph_count(complex_graph: nx.Graph) -> nx.Graph:
     return simple_graph
 
 
+def simplify_graph(G, strict=True, remove_rings=True, track_merged=False):
```

Comment:
Add type hints and avoid defining default values (make the caller explicitly state which value they are providing)

Reply:
please defer from using single variable names (specially in caps), rather use something explicit (for instance `graph_to_simplify`)

Reply:
Why is this here and not in the OsmnxWrapper?